src/main/java/build/buildfarm/Cat.java

```diff
@@ -604,6 +607,95 @@ static int deadlineSecondsForType(String type) {
     return 10;
   }
 
+  private static void getWorkerProfile(Instance instance) {
+    WorkerProfileMessage response = null;
+    try {
+      response = instance.getWorkerProfile();
+    } catch (StatusRuntimeException e) {
+      System.out.println(e.getMessage());
+    }
+    String strNumFormat = "%-50s : %d";
+    System.out.println(
+        String.format(strNumFormat, "Current Entry Count", response.getCASEntryCount()));
```

Comment:
You're going to NPE here if you don't get a response and the exception occurs above - easier just to not catch it here.